fineract-loan/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/LoanSummary.java

```diff
@@ -225,7 +238,7 @@ public void updateSummary(final MonetaryCurrency currency, final Money principal
         this.totalPenaltyChargesOutstanding = totalPenaltyChargesCharged.minus(this.totalPenaltyChargesRepaid)
                 .minus(this.totalPenaltyChargesWaived).minus(this.totalPenaltyChargesWrittenOff).getAmount();
 
-        final Money totalExpectedRepayment = Money.of(currency, this.totalPrincipalDisbursed).plus(this.totalInterestCharged)
+        final Money totalExpectedRepayment = Money.of(currency, this.totalPrincipalOutstanding).plus(this.totalInterestCharged)
```

Comment:
This looks incorrect... outstanding always changing, but total expected repayment does not....